sunpy/map/sources/hinode.py

```diff
@@ -34,6 +49,9 @@ def __init__(self, data, header, **kwargs):
 
         GenericMap.__init__(self, data, header, **kwargs)
 
+        # converting data array to masked array
+        #self.data = ma.masked_where(self.data > SATURATION_LIMIT, self.data)
```

Comment:
why is this commented out? If we are not going to use it, why is the constant still being defined?